Combine.playground/Pages/Foundation and Combine.xcplaygroundpage/Contents.swift

```diff
@@ -17,15 +19,25 @@ URLSession.shared.dataTaskPublisher(for: URL(string: "https://www.avanderlee.com
     .decode(type: DecodableExample.self, decoder: JSONDecoder())
 
 /*:
- ##### A Publisher for notifications
+ ### A Publisher for notifications
  */
 NotificationCenter.default.publisher(for: .NSSystemClockDidChange)
 
 /*:
- ##### KeyPath binding to NSObject instances
+ ### KeyPath binding to NSObject instances
  */
 let ageLabel = UILabel()
 Just(28)
     .map { "Age is \($0)" }
     .assign(to: \.text, on: ageLabel)
+
+/*:
+### A Timer publisher exposing Cocoa's `Timer`
+- this one is a bit special as it is a `Connectable`
+- ... use `autoconnect` to automatically start it when a subscriber subscribes
+*/
+let publisher = Timer
+	.publish(every: 1.0, on: .main, in: .common)
+	.autoconnect()
```

Comment:
It might be good to add some documentation on `autoconnect` as well to explain what it does